### DISCUSSION TITLE
Welcome to Universal Video Maximizer Discussions!

### DISCUSSION THREAD

**Discussion by trophygeek** [trophygeek/universalvideomaximizer#68] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ishigamii** [comment]
Hi there thanks for your lovely extension I am loving this. 
I just found out about the shortcut to use it and it's nice the only "problem" is that when I desactivate it using the same shortcut to put the video back to it's original space and size, it doesn't do it and instead shows the menu between speed (when the advance feature is on) would be nice to have them separated maybe in two differents shortcuts (don't know if possible or not easily) 
Thanks again 👍

**Reply by trophygeek** [reply]
Thanks for the feedback and you're making a great observation.

In general, that speed dropdown behavior isn't ideal. But it's that way because of how the Chrome extension permission requests work and the fact that browser shortcut keys are controlled by Chrome.

But here are two things you can try.

A. Escape key.
When the video is maximized, there's a quick restore by pressing the Escape key.
This has some issues because it page context does it and the Extension doesn't realize it's happened. So the extension thinks it's still zoomed.

Not ideal, but is might solve your issue?

B. Switch extension to Toggle mode.
There is a work-around: 
1. Go into Video Maximizer's Options by right clicking the extension icon and selecting Options.
2. Disable the "Advanced playback features".

This causes the maximize to toggle when the extension is clicked, but it also removes the ability to show speeds in that popup menu.

**Reply by ishigamii** [reply]
Thanks indeed escape works sometimes :D I disabled the  advance to have it easier for the moment
